### DISCUSSION TITLE
OGB model documentation

### DISCUSSION THREAD

**Discussion by Percefoni** [snap-stanford/ogb#398] [General]
Good evening. I have a question , regarding your code for the ogb wikidata. I've been working on graph neural networks lately, but because it's something new to me I'm having difficulties. Could I find somewhere documentation for the code and more specifically for the methods.Thanks in advance.

**Comment by weihua916** [comment]
Hi! I do not think GNNs are used in wikidata knowledge graphs. For knowledge graphs, the best-performing models are knowledge graph embeddings.

**Comment by Percefoni** [comment]
Thank you for your answer. I guess my question was not so clear.Is there a detailed commentary of the code(model.py) somewhere?
